src/generators/rust/RustDependencyManager.ts

```diff
@@ -0,0 +1,12 @@
+import { AbstractDependencyManager } from '../AbstractDependencyManager';
+import { RustOptions } from './RustGenerator';
+
+export class RustDependencyManager extends AbstractDependencyManager {
```

Comment:
This can be very diverse in what it can be used for, for example in TypeScript we added support functions to help better add dependencies based on the different module systems 🙂 https://github.com/asyncapi/modelina/blob/bfe8d78df9db61f467c1f0aee6e2d4d4d1b8f7b7/src/generators/typescript/TypeScriptDependencyManager.ts#L32

Reply:
I also imagine that in the future, we can have it include package imports etc, to dynamically add dependency libraries for the support files. However, that is for the future if it's ever requested 🙂